implementations.py

```diff
@@ -2,21 +2,23 @@
 import sys
 
 # add your QUIC implementation here
-IMPLEMENTATIONS = {  # name => [ docker image, role ]; role: 0 == 'client', 1 == 'server', 2 == both
```

Comment:
Why remove this comment? It seems useful for folks adding implementations -- they don't have to go digging. Maybe move this out to a comment above the IMPLEMENTATIONS definition ?

Reply:
It was just moved to the line below.